### PR TITLE
patch: Cms insurance work for 702 by SN

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #

### CODE REVIEW

File: library/js/xl/jquery-datetimepicker-2-5-4-translated.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        , scrollMonth: false
+        , rtl: (languageDirection === 'rtl') ? true : false
+    };
+    defaults = Object.assign(defaults, params);
+    // go through and setup more specific settings
+    if (params.timepicker) {
+        if (params.showSeconds) {
+            if (params.formatInput) {
+                defaults.format = DateFormatRead("jquery-datetimepicker") + ' H:i:s';
+            }
```
Comment (bradymiller) [inline library/js/xl/jquery-datetimepicker-2-5-4-translated.js:85]: need to include this fix:
https://github.com/openemr/openemr/pull/7223/files

File: templates/patient/insurance/_insurance_edit_screen_edit.html.twig [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+                    <option value="">{{ 'Unassigned'|xlt }}</option>
+                </select>
+                <a data-modal-title="{{ "Insurance Search/Add/Edit"|xla }}" class="medium_modal btn btn-primary insurance-search-open-finder" href="../../practice/ins_search.php?ins=" role="button">{{ 'Search/Add/Edit'|xlt  }}</a>
+            </div>
+        </div>
+        {% for fieldRow in fields.left %}
+            {% if fieldRow.skip is not defined or fieldRow.skip %}
+                <div class="form-row"><!-- start nested row -->
+                    <div class="col-md-3 pb-1 label_custom">
+                        <span data-validation-field="{{ fieldRow.field_id }}" class='{%if fieldRow.required is defined and fieldRow.required %}required{% endif %}'>{{ fieldRow.name|text }}</span>
```
Comment (bradymiller) [inline templates/patient/insurance/_insurance_edit_screen_edit.html.twig:94]: looks like fieldRow.field_id needs attr

**Comment by bradymiller** [comment]
dance-bear

**Comment by stephenwaite** [comment]
bringing in, thanks for the review @bradymiller , super work @adunsulag !

southpark_cheering

**Comment by adunsulag** [comment]
@stephenwaite I'm good with merging this in.  Do we need to wait on anything else here?

**Pull request merged**